use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,217 @@
+# How To Pass a Physical NIC Into a Kata Container
+
+* [Before you start (Restrictions, Requirements and Assumptions)](#before-you-start-restrictions-requirements-and-assumptions)
+* [Pass a physical NIC into a Kata Container](#Pass-a-physical-NIC-into-a-Kata-Container)
+  * [Part 1 – The Host](#part-1--the-host)
+  * [Part 2 – The Container](#part-2--the-container)
+* [Test Your Setup](#Test-Your-Setup)
+
+![](./images/NIC%20passthrough%20Diagram.png)
+
+
+In this guide we walk through the process of passing a physical NIC into a KC (Kata Container). The "usual" way that a KC is wired for communication is through a series of bridges and virtual NICs as can be seen in the Image above (marked as default). 
```

Comment:
/Image/image/
I would rewrite it as a "through a bridge and virtual NIC" and drop the `a series..`

Reply:
Done.